java/runtime/src/main/java/io/ray/runtime/placementgroup/PlacementGroupId.java

```diff
@@ -0,0 +1,58 @@
+package io.ray.runtime.placementgroup;
+
+import io.ray.api.id.BaseId;
+import java.io.Serializable;
+import java.nio.ByteBuffer;
+import java.util.Arrays;
+import java.util.Random;
+
+/**
+ * Represents the id of a placement group.
+ */
+public class PlacementGroupId extends BaseId implements Serializable {
```

Comment:
It seems that we don't need this in Java at all? Java can just use a byte[] id?

Reply:
The length of PlacementGroupId must be 16. Use PlacementGroupId instead of byte[] is better.